persistence/sql/migrations/sql/20241023142500000001_drop_unused_indices_identity_credentials.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE INDEX identity_credentials_nid_identity_id_v2_idx ON identity_credentials (nid ASC, identity_id ASC);
```

Comment:
I think we might be missing a secondary index that has identity_id as the first piece, which will sped up account deletion.